source/assets/critical/c_landingpage.scss

```diff
@@ -3,3 +3,218 @@
 @import 'global';
 @import 'header';
 @import 'footer';
+@import '../../_patterns/variables';
+@import '../../_patterns/breakpoints';
+@import '../../_patterns/mixins';
+@import '../../_patterns/10-atoms/indicators/shimmer';
+@import '../../_patterns/40-templates/landingpage';
+
+
+
+//PASCALS STUFF
+.loading__only {
```

Comment:
die "loading__only"-klasse könnte man für die konsistenz im fef in "h-loading-only" umbenennen